adamacs/behavior.py

```diff
@@ -113,8 +93,50 @@ class TrialVariable(dj.Part):
 		variable_value: varchar(2000)
 		"""
 
+@schema
+class BehaviorIngest(dj.Imported):
+	definition = """
+	-> session.Recording
+	"""
+  @property
+  def key_source(self): #opional means of subsetting or reordering keys worth ingesting
+  	good_records = session.Recording.fetch('KEY') #& session.Qualitycontrol
+  	return good_records
+
+	def make(self, key): # reading bpod data to populate
+		# could model dir navigation after element_array_ephys
+		# which uses config file for root dir and csv for relative paths
+		# https://github.com/datajoint/workflow-array-ephys/blob/main/workflow_array_ephys/paths.py
+		bpod_root_dir = pathlib.Path(get_beh_root_dir(key))
+		bpod_sess_dir = pathlib.Path(get_beh_sess_dir(key))
+		bpod_dir = find_full_path(bpod_root_dir,bpod_sess_dir)
+
+		for file in os.listdir(bpod_dir): if filename.endswith(".mat"):
+			trial_info = load_bpod_matfile(key, bpod_dir + file)
```

Comment:
```suggestion
		bpod_filepath = next(bpod_dir.glob('*.mat'))
		trial_info = load_bpod_matfile(key, bpod_filepath )
```

Reply:
Let's confirm if checking by just an extension of `.mat` is sufficient to identify the bpod mat file. Could there be other `.mat` files in the directory?

Reply:
Certainly. Digging through the shared example data, there were other filetypes in the same directory (.m, .fig), but all `.mat` were recordings. I'll confirm this explicitly at our meeting.